requests_cache/backends/base.py

```diff
@@ -105,11 +105,7 @@ def remove_old_entries(self, created_before):
         """ Deletes entries from cache with creation time older than ``created_before``
         """
         keys_to_delete = set()
-        for key in self.responses:
-            try:
-                response, created_at = self.responses[key]
-            except KeyError:
-                continue
+        for key, (response, created_at) in self.responses.items():
```

Comment:
I think it will not work in Python 3, because ``items()`` is an iterator. Proper fix will be 
```
 for key in list(self.responses):
```